lsp/src/test/scala/org/ensime/lsp/rpc/MessageCompanionsSpec.scala

```diff
@@ -29,6 +28,9 @@ object MessageCompanionsSpec {
   }
 
   object Commands extends DefaultJsonProtocol with FamilyFormats {
+    override implicit def optionFormat[T: JsonFormat]: JsonFormat[Option[T]] =
```

Comment:
It looks like a boilerplate :thinking:

Reply:
This is the cost of the mixins. This problem goes away when moving to export-hook style (next on the list)